### DISCUSSION TITLE
OTLP Logger for NodeJS

### DISCUSSION THREAD

**Discussion by derN3rd** [SigNoz/signoz#1546] [Q&A]
Hey there, 

we are currently trying out to use signoz with opentelemetry implementations on our server side, which seems to work just fine.
WIth the newest release (0.11.0) log support was added, but I couldn't find an implementation for this either on the js libs of opentelemetry nor similar.

One solution would be to add a new transportation for our logger (winston), but I wanted to ask here first if someone has the same "issue" and found a good and simple solution.

Thanks in advance

**Comment by srikanthccv** [comment]
There exists an OTEL instrumentation library for [`winston`](https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/plugins/node/opentelemetry-instrumentation-winston) but as of today it only supports the log enrichment with trace context information via hooks. You should be able to use console transport + fluent (or some other log collection agent) to send data to SigNoz. Eventually, OpenTelemetry will provide an HTTP transport library that works with the OTLP protocol. And this user guide should help you with the different mechanisms of collecting and sending logs to SigNoz https://signoz.io/docs/userguide/logs/#collecting-logs.
